### PR TITLE
ME-1671 Fileserver look and feel

### PR DESCRIPTION
**Pull request by atoonk**
# Description

Improved look and feel for fileserver 

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

# Checklist:

- [x] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Review by th3wingman — APPROVED** [review]
LGTM

File: internal/http/server.go [hunk 1]
```diff
@@ -98,6 +102,229 @@ func StartLocalHTTPServer(dir string, l net.Listener) error {
 	return nil
 }
 
+func myFileServer(dir string) http.HandlerFunc {
```
Comment (adrianosela) [inline internal/http/server.go:105]: Rename to customFileServer

File: internal/http/server.go [hunk 2]
```diff
@@ -98,6 +102,229 @@ func StartLocalHTTPServer(dir string, l net.Listener) error {
...
+	fs := http.Dir(dir)
+
+	return func(w http.ResponseWriter, r *http.Request) {
+		path := r.URL.Path
+
+		if !strings.HasPrefix(path, "/") {
+			path = "/" + path
```
Comment (adrianosela) [inline internal/http/server.go:112]: Use format string over string addition

File: internal/http/server.go [hunk 3]
```diff
@@ -98,6 +102,229 @@ func StartLocalHTTPServer(dir string, l net.Listener) error {
...
+					}
+				})
+			}
+
+			sort := r.URL.Query().Get("sort")
+			dir := r.URL.Query().Get("dir")
+			lastSort := r.URL.Query().Get("last_sort")
+			lastDir := r.URL.Query().Get("last_dir")
+
+			fmt.Fprint(w, `<html><head>
```
Comment (adrianosela) [inline internal/http/server.go:191]: Use const for the multiline string at the top, making it easier to edit

**Review by adrianosela — APPROVED** [review]
Nice, nits left

**Pull request merged**